core/lib/dal/Cargo.toml

```diff
@@ -29,22 +29,21 @@ url = "2"
 prost = "0.12.1"
 rand = "0.8"
 tokio = { version = "1", features = ["full"] }
-sqlx = { version = "0.5.13", default-features = false, features = [
+sqlx = { version = "0.7.3", default-features = false, features = [
     "runtime-tokio-native-tls",
```

Comment:
Nit: Judging by [the crate readme](https://crates.io/crates/sqlx), `runtime-tokio-native-tls` is soft-deprecated in favor of `runtime-tokio` + `tls-native-tls`.

Reply:
Changed to those, any particular reason we don't use `rustls` here btw?

Reply:
I vaguely remember some annoying issue that didn't let us do that back in the time, but don't even remember what it was.
Maybe it's even long resolved.